File: Python/007_rectangle.py
# 5. Object-Oriented Programming
# Task 8: Define a class named Rectangle that can be used to create 
# rectangle objects. Include an initializer, properties for width and 
# height, and a method to calculate the area of the rectangle. 
# Create several rectangle objects, calculate their area, and 
# print the results.
class Rectangle:

    def __init__(self, width, height):
        self.width = width
        self.height = height
    
    def rectangles(self):
        for y in range(self.height): # controls columns
            print("*"*self.width)
            
    def area(self):
        area = int(self.height * (self.width))
        print(f"The area of the rectangle is {str(area)}")
            

if __name__ == "__main__":
    value1 = int(input("enter width: "))
    value2 = int(input("enter height: "))
    myRect = Rectangle(value1, value2)
    myRect.rectangles()
    myRect.area()

File: JavaScript/007_rectangle.js
const prompt = require("prompt-sync")({sigint:true});


class Rectangle {
    constructor(width, height) {
        this.width = width;
        this.height = height;
    }
    area() {
        let area = Number(this.height * (this.width));
        console.log(`The area of the rectangle is ${area}`);
    }
    rectangle() {
        for (let i = 0; i < this.height; i++) {
            console.log("*".repeat(this.width));
        }
    }
}

function main() {
    let a = prompt("enter width: ");
    let b = prompt("enter height: ");
    let myTest = new Rectangle(a, b);
    myTest.rectangle();
    myTest.area()
}

main()